packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -7,6 +7,11 @@ import Inquirer from 'inquirer'
 import getChoices from '../../helpers/choices'
 import {formatHookErrorMsg, hookFailed} from '../../helpers/hook-error'
 
+const UI_FRAMEWORK_OPTIONS: { [key: string]: string } = {
```

Comment:
Should we create a folder `template` or `better_name` to contain the code for each add-on e.g. `uiFramework.ts`, `versionControl.ts`. Then this file won't get too big later 🤔

Reply:
I have a suggestion in this PR: https://github.com/nimblehq/react-templates/pull/108/files
(I'll edit the description with some insights about what changed) 
Just for demo purpose :)

Reply:
Ok, implemented here: 7556e33 :)